seismic/xcorqc/xcorqc.py

```diff
@@ -236,6 +253,16 @@ def xcorr2(tr1, tr2, sta1_inv=None, sta2_inv=None,
                     tr2_d = bandpass(tr2_d, flo, fhi, sr2, corners=6, zerophase=True)
                 # end if
 
+                # resample
+                if(resample_rate):
```

Comment:
I think there should be some checks that the `resample_rate` is at least the Nyquist frequency of `fhi`, otherwise aliasing could easily corrupt the signal.

Reply:
Added a parameter consistency check at a higher level to make sure this doesn't happen.